pkg/api/metrics.go

```diff
@@ -40,6 +44,101 @@ func (hs *HTTPServer) QueryMetricsV2(c *models.ReqContext) response.Response {
 	return toJsonStreamingResponse(resp)
 }
 
+func checkDashboardAndPanel(ctx context.Context, dashboardId, panelId string) error {
+	if dashboardId == "" || panelId == "" {
+		return models.ErrDashboardOrPanelIdentifierNotSet
+	}
+
+	id, err := strconv.ParseInt(dashboardId, 10, 64)
+	if err != nil {
+		return models.ErrDashboardIdentifierInvalid
+	}
+
+	query := models.GetDashboardQuery{
+		Id: id,
+	}
+
+	if err := bus.Dispatch(ctx, &query); err != nil {
```

Comment:
We are working on removing the bus. Can the SQL store be passed in instead (property on `hs *HTTPServer`) and its `GetDashboard` method be used instead?